FilePersistence/src/version_control/Commit.cpp

```diff
@@ -103,4 +103,60 @@ bool Commit::getFileContent(QString fileName, const char*& content, int& content
 		return false;
 }
 
+QStringList Commit::nodeLinesFromId(Model::NodeIdType Id, bool findChildrenByParentId) const
+{
+	auto id =  Id.toString();
+	QStringList matches;
+	for (auto file : files())
+	{
+		int indexOfId = 0;
+		auto content = QString::fromUtf8(file->content());
+
+		indexOfId = content.indexOf(id, indexOfId);
+		if (indexOfId != -1)
+		{
+			int invalid = 0;
```

Comment:
this should be a `bool` not an int. You only ever use 0 and 1. So use, `true`, and `false` instead.